juniper_official/routing/collect-sr-color-lsp-stats.rule

```diff
@@ -83,16 +86,28 @@ healthbot {
                 }
                 type integer;
                 description "LSP packet rate";
+            }            
+            field device-vendor {
+                constant {
+                    value "{{inp-device-vendor}}";
+                }
+                type string;                
+                description "Indicates device vendor";
             }
-            variable input-to-address {
+            variable inp-lsp-destination {
                 value .*;
                 description "SR color LSP destination address to monitor, regular expression, eg '11.0.0.*|10.52.135.*'";
                 type string;
             }
-            variable input-color {
+            variable inp-lsp-color {
```

Comment:
Changing of variable names will cause upgrade failures if the rules is already instantiated on older release